docs/source/using_api.rst

```diff
@@ -114,3 +114,23 @@ it from the menu in top right corner.
 
 To store the reason for the change, add HTTP header ``PDC-Change-Comment``,
 whose value is an arbitrary string that will be stored with the change.
+
+
+Override Ordering
+-----------------
+
+With OrderingFilter class, the users can override the ordering of the results
+with query parameter. By default, the query parameter is named ``ordering``. And
+the ``ordering_fields``, which by using the special value ``__all__``, means that
+a view allow ordering on any ``model`` field or queryset aggregate.
```

Comment:
I think customer should no care about what is `model`. It's internal implementation, customer also don't know what fields in a model. Also as discussed, should we use model's fields?